### PR TITLE
[docs] Docs Metadata 

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
//cc @AndreyBelym

### CODE REVIEW

File: docs/articles/documentation/test-api/test-code-structure.md [hunk 1]
```diff
@@ -277,6 +281,54 @@ fixture `MyFixture`
     .page `../my-project/index.html`;
 ```
 
+## Specifying Testing Metadata
+
+TestCafe allows you to specify additional information for fixtures and tests in the form of *field-value metadata*.
```
Comment (AndreyBelym) [inline docs/articles/documentation/test-api/test-code-structure.md]: I think right term is *key-value*.

File: docs/articles/documentation/test-api/test-code-structure.md [hunk 2]
```diff
@@ -277,6 +281,54 @@ fixture `MyFixture`
...
+
+### Fixture Metadata
+
+You can specify metadata for a fixture by using the `meta` method in the [fixture declaration](#fixtures).
+
+```js
+fixture.meta({ field1: 'value1', field2: 'value2', field3: 'value3' });
```
Comment (AndreyBelym) [inline docs/articles/documentation/test-api/test-code-structure.md]: Method has an additional overload:
`.meta(name, value)`

Parameter  | Type   | Description
---------- | ------ | -----------------
`name` | string | Name of the metadata entry
`value` | string | Value of the metadata entry

File: docs/articles/documentation/test-api/test-code-structure.md [hunk 3]
```diff
@@ -277,6 +281,54 @@ fixture `MyFixture`
...
+
+```js
+fixture 'MyFixture'
+    .meta('fixtureID', 'f-0001')
+    .meta({ author: 'John', creationDate: '05/03/2018' });
+```
+
+### Test Metadata
+
+You can specify metadata for an individual test by using the `test.meta` method.
```
Comment (AndreyBelym) [inline docs/articles/documentation/test-api/test-code-structure.md]: We also should note, that `.meta()` calls override values for entries specified in previous `.meta()` calls and it doesn't matter if it was `fixture.meta()` or `test.meta()` call. All metadata entries specified for a fixture are available in tests that reside in the fixture.

**Comment by MargaritaLoseva** [comment]
fpr @AndreyBelym @VasilyStrelyaev

File: docs/articles/documentation/test-api/test-code-structure.md [hunk 4]
```diff
@@ -277,6 +279,74 @@ fixture `MyFixture`
...
+    ```js
+    test.meta('key2', 'value2')
+    ```
+
+    Parameter | Type   | Description
+    --------- | ------ | -----------------
+    `name`    | String | The name of the metadata entry
+    `value`   | String | The value of the metadata entry
+
+* You can specify a set of metadata entries for a fixture or a test.
```
Comment (VasilyStrelyaev) [inline docs/articles/documentation/test-api/test-code-structure.md]: I don't like these two points:

```
* You can specify one metadata entry for a fixture or a test.
* You can specify a set of metadata entries for a fixture or a test.
```

After reading the first one, I begin thinking that I can specify **only** one entry. I personally got surprised. And then I see the second point that contradicts the first one.

Let's either add `either`...

```
* You can either specify one metadata entry for a fixture or a test...
* ...Or a set of metadata entries.
```

**(but that's not quite good)**

... or add a phrase before this list.

```
To specify metadata, use the `meta` method. You can call this method both for a fixture and a test.

The 'meta' method allows you to specify one or several meta entries.

* Specifying one metadata entry for a fixture or a test.

blablabla

* Specifying a set of metadata entries for a fixture or a test.

blablabla
```

or even 

```
### Specifying one metadata entry for a fixture or a test

blablabla

### Specifying a set of metadata entries for a fixture or a test

blablabla
```

**Comment by MargaritaLoseva** [comment]
fpr

**Comment by MargaritaLoseva** [comment]
@dirk-pieterse Please review these changes -- https://github.com/DevExpress/testcafe/pull/2372/files#diff-6b4328e517bb70724f546fb3796f8436

**Pull request merged**